src/SignalR/clients/ts/FunctionalTests/package.json

```diff
@@ -30,9 +29,11 @@
     "karma-sauce-launcher": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-summary-reporter": "^1.6.0",
+    "process": "^0.11.10",
     "rxjs": "^6.3.3",
     "ts-node": "^8.6.2",
     "typescript": "^3.7.5",
```

Comment:
you could use the ^4.1.5 version for typescript

Reply:
Done

Reply:
Things are going faster. You could even use ^4.2.0 version of typescript which is just released.